### PR TITLE
Include setup.py

### PR DESCRIPTION
**Pull request by sudeepnarala**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(
+	name='sibispy',
+	version='1.0',
+	py_modules=["sibispy"]
+)
```
Comment (jimklo) [inline setup.py:7]: I'd like to see the various entrypoints and scripts from the `cmd` directory get added to this.
Also missing dependencies.
see `https://github.com/sibis-platform/hivalc-data-integration/blob/master/setup.py` as an example.
Also if we can get the tests declared in the setup, that would be good.  We may need to discuss this a bit as the sibispy requires quite a bit of configuration to work, namely a `~/.sibis-general-config.yml` and a mapping to an `operations` directory that stores `sibis-test-config.yml` and `sibis-sys-config.yml` and `special-cases.yml`.

**Review by jimklo — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (sudeepnarala) [reply setup.py:32]: I added the entry points. We can work on the setup and get everything fixed.

**Comment by sudeepnarala** [comment]
I added the entry points. We can work on the setup and get everything fixed.

**Comment by jimklo** [comment]
Okay, so a things you should do next, which you can do on `sibis-storage`, is to:

1. update the docker containers to use sibispy as a package
  - `pipeline-core` : probably remove `/sibis-software/python-path` from the `PYTHON_PATH`
  - `pipeline-system`: have it `pip install` sibispy.
  - `pipeline-main`: update to use newer `pipeline-system`.

2. Create a container instance; probably similar to what's defined in either `hivalc-data-integration/docker/pipeline-back` or `ncanda-data-integration/docker/pipeline-back`.  Ncanda might be simpler to start.  Then let's make sure the tests in sibispy can be run and pass using the updated `pipeline-main`.

**Comment by kipohl** [comment]
still needs to be worked on

**Comment by jimklo** [comment]
Closing for now. Will revisit later.

**Pull request closed without merging**